packages/webpack-common/src/extractText.js

```diff
@@ -18,7 +18,7 @@ export {
  */
 function getLoaderConfigByType (context, webpackConfig, fileType) {
   const loaderConfig = webpackConfig.module.loaders.find(
-    (loader) => loader.test === context.fileType(fileType)
+    (loader) => regexEqual(loader.test, context.fileType(fileType))
```

Comment:
I think it is because of the deep clone webpack-merge is doing behind the scene. In a strict functional way, it is expected that the Regex instance might be a different one with the same value. I'm not sure it is bad to use a more reliable way to compare regexes. I don't think we can expect two instances with the same value on different loaders. I don't think regex are seen as primitive types in javascript.